src/index.ts

```diff
@@ -55,6 +55,23 @@ function getBody(payload: WebhookPayload) {
   }
 }
 
+function formatDuration(timeCompleted: string, timeStarted: string) {
+  let durationMs = new Date(timeCompleted).getTime() - new Date(timeStarted).getTime();
```

Comment:
Subtracting two dates already gives you a duration in ms, rendering the `getTime` unnecessary

Reply:
Without getTime how would I get the value in the object ??

Reply:
This works for me
`let durationMs = Date.parse(timeCompleted) - Date.parse(timeStarted);`